en/docs/quick-starts/android-sample.md

```diff
@@ -1,49 +1,38 @@
-## Authenticate to Android Sample Application
+# Authenticate to Android Sample Application
 
-< Intro - to the Sample >
+## Register Application
 
-## Overview
- < Explain the use case with diagrams - can be links to concepts > 
+ 1. Expand **Inbound Authentication Configuration > OAuth/OpenID Connect Configuration** and click **Configure**.
  
- < Image explaining the scenario>
+ 2. Enter the **Callback URL** and click **Add**. 
  
-## Setup Sample
-
-
-## Register Application
-
- < Description on sample android mobile app specific config >
-
-{!fragments/oauth-app-config-basic.md!}
-
-
-    | Field                 | Value         | 
-    | --------------------- | ------------- | 
-    | Service Provider Name | sample-app  |
-    | Description           | This is a mobile application  | 
-    | Call Back Url         | wso2sample://oauth2  | 
-
+     The **Callback URL** is the exact location in the service provider's application where an access token would be sent. This URL should be the redirect scheme 
+     of the application that the user is redirected to after successful authentication.
+     
+ 3. To configure more advanced configurations, see [OAuth/OpenID Connect Configurations](../../login/oauth-app-config-advanced). 
+ 
+ 4. Note the **OAuth Client Key** that appear. 
+ 
+ 
+     | Field                 | Value         | 
+     | --------------------- | ------------- | 
+     | Service Provider Name | sample-app  |
+     | Description           | This is a mobile application  | 
+     | Call Back Url         | wso2sample://oauth2  | 
+     
 5. Enable following properties:
 - PKCE Mandatory
 - Allow authentication without the client secret
 
-## Configure the Android SDK
-
-To Build the SDK locally:
+## Setup the sample
 
-1. Clone this project: https://github.com/wso2-extensions/identity-sdks-android.git
-    `git clone https://github.com/wso2-extensions/identity-sdks-android.git`
 
-2. Build the library in your local maven. Run the following commands. Now the library artifact will be
- available in your local .m2 cache. 
-    - `./gradlew clean assembleRelease`
-    - `./gradlew publishToMavenLocal `
+1. Download the android sample by [clicking here](https://github.com/wso2-extensions/identity-samples-android/archive/master.zip)
```

Comment:
Download the [android sample](https://github.com/wso2-extensions/identity-samples-android/archive/master.zip).